practices/oocss.md

```diff
@@ -1,12 +1,39 @@
 # Object Oriented CSS
 
+
+
 ## What is OOCSS
 
+OOCSS or 'Object Orientated CSS' is an approach to writing css that aims to encourage reusable, maintainable and predictable css. It has spawned a number of different approaches to standardise the concept most of which adhere to the core principles of OOCSS outline below.
+
+A CSS 'object' is a repeating visual pattern, that can be abstracted into an independent snippet of HTML, CSS, and possibly JavaScript. That object can then be reused throughout a project. The aim is to break down the css declaration into the smallest blocks possible in order to maximise their potential for reuse.
```

Comment:
would like to change to "smallest blocks necessary to enable reuse" rather than "smallest blocks possible in order to maximise their potential for reuse". The smallest blocks _possible_ is the aim of atomic CSS, not OOCSS as a whole.